src/appkit.cs

```diff
@@ -18755,6 +18786,7 @@ partial interface NSTextView : NSTextInputClient, NSTextLayoutOrientationProvide
 		[Export ("lowerBaseline:")]
 		void LowerBaseline (NSObject sender);
 
+		[Deprecated (PlatformName.MacOSX, 10, 11, message: "Use unicode characters via the character palette.")]
```

Comment:
Mmh the character palette? Can we be more specific?

Reply:
```/11x/AppKit.framework/Headers/NSTextView.h:- (void)toggleTraditionalCharacterShape:(nullable id)sender NS_DEPRECATED_MAC(10_0, 10_11, "Use the traditional shaped characters encoded in the Unicode standard. Access the characters via the character palette.");
```